master/docs/manual/cfg-builders.rst

```diff
@@ -152,7 +152,8 @@ If ``collapseRequests`` is given, it completely overrides the global configurati
 Possible values for both ``collapseRequests`` configurations are:
 
 ``True``
-    Requests will be collapsed if their sourcestamp are compatible (see below for definition of compatible).
+    Requests will be collapsed if they and their sourcestamps are
```

Comment:
do not break one-sentence-per-line please

Reply:
Had not heard of that rule, I'll fix.

Reply:
Hmm... Then we need to fix it.

When changes are following this policy, it's much easier to review the changes (especially "typos").

Reply:
Oh, found it: http://docs.buildbot.net/latest/developer/style.html#documentation